sdk/test-utils/recorder-new/src/recorderRequestModifier.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { isPlaybackMode, isRecordMode } from "@azure-tools/test-recorder";
+import { WebResourceLike } from "@azure/core-http";
+import { PipelineRequest } from "@azure/core-rest-pipeline";
+import { RecorderError } from "./utils/utils";
+
+export class RecorderRequestModifier {
+  constructor(public mode: string, private url: string) {}
+  /**
+   * For core-v1 (core-http)
+   */
+  redirectRequest(request: WebResourceLike, recordingId?: string): void;
+
+  /**
+   * For core-v2 (core-rest-pipeline)
+   */
+  redirectRequest(request: PipelineRequest, recordingId?: string): void;
```

Comment:
I don't see this overload being used.

Reply:
`modifyRequest` internally class this

Reply:
removed the file.
rolled back to being private members